ktx-library-codelab/build.gradle

```
@@ -15,7 +15,34 @@
  */
 
 buildscript {
-    ext.kotlin_version = '1.3.60'
+    ext.kotlin_version = '1.4.21'
+    ext {
+        def appcompat_ver = "1.2.0"
```

```suggestion
        def appcompat_version = "1.2.0"
```

Replaced these for `_version`? It's weird to see `kotlin_version` and then other things here